### PR TITLE
[tbb] Change hwloc into a default feature

### PR DESCRIPTION
**Pull request by ilya-lavrenov**
**Changes:**
- Added `hwloc` as a default feature for TBB. This feature enables efficient Hybrid processors support, which is are quite popular nowadays.
- Note, that this capability is enabled by default in all other package managers like apt, conda, Conan, brew and so on.

### CODE REVIEW

File: ports/tbb/vcpkg.json [hunk 1]
```diff
@@ -17,6 +17,12 @@
       "host": true
     }
   ],
+  "default-features": [
+    {
+      "name": "hwloc",
+      "platform": "!static & !osx"
```
Comment (LilyWangLL) [inline ports/tbb/vcpkg.json:23]:
```suggestion
      "name": "hwloc"
```
The following codes has `"supports": "!static & !osx",`.

**Review by LilyWangLL — CHANGES_REQUESTED** [review]

File: ports/tbb/vcpkg.json [hunk 2]
```diff
@@ -17,6 +17,12 @@
...
+    }
```
Comment (LilyWangLL) [inline ports/tbb/vcpkg.json:24]:
```suggestion
    "name": "hwloc"
```
The following codes has `"supports": "!static & !osx",`.

**Review by LilyWangLL — CHANGES_REQUESTED** [review]

File: ports/tbb/vcpkg.json (hunk 2 above)
Comment (dg0yt) [reply ports/tbb/vcpkg.json:24]: ... and that's why it must have `platform` here!

**Review by LilyWangLL — DISMISSED** [review]

**Review by BillyONeal — APPROVED** [review]

**Comment by BillyONeal** [comment]
Thanks!

**Pull request merged**